endoreg_db/models/media/video/video_file.py

```
@@ -1,6 +1,7 @@
 """Concrete model for video files, handling both raw and processed states."""
 
 import logging
+from os import pipe
```

Ruff (0.8.2)

`os.pipe` imported but unused

Remove unused import: `os.pipe`

(F401)